### ISSUE TITLE
disabled navigation button labels

### ISSUE THREAD

**Issue by danielghost** [cgkineo/adapt-scrollSnap#44]
When using a screen reader, disabled navigation buttons are just read as "button unavailable". Whilst the label is rendered, this is ignored when marked as disabled, and requires an `aria-label`.

**Comment by oliverfoster** [comment]
Which screen reader/browser? Why is this a problem? How would an `aria-label` change what is read? Accessible name calculations should be performed in the same way regardless. 

https://www.w3.org/TR/accname-1.2/#:~:text=The%20accessible%20name%20and%20description,specific%20to%20individual%20content%20languages.

**Comment by danielghost** [comment]
At one point it didn't read a label for the button at all when disabled, so a screen reader user wouldn't know which button was being referred to via "button unavailable". This was flagged as an issue in a project and required the above fix, but it no longer seems to be the case with the buttons working as expected without the addition of the `aria-label`.
